runtime/gc_realtime/WorkPacketsRealtime.cpp

```diff
@@ -21,19 +20,16 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
-
-#include "j9.h"
-#include "j9protos.h"
-#include "j9port.h"
+#include "omr.h"
+#include "omrcfg.h"
 #include "omrthread.h"
 
-#include "Debug.hpp"
 #include "EnvironmentRealtime.hpp"
-#include "GCExtensions.hpp"
+#include "GCExtensionsBase.hpp"
 #include "IncrementalOverflow.hpp"
 #include "IncrementalParallelTask.hpp"
-#include "RememberedSetSATB.hpp"
```

Comment:
Well, if you remove this line you should add `#include "WorkPacketsSATB.hpp"` explicitly here

Reply:
ok, never mind. I missed the fact that MM_WorkPacketsSATB is it's parent.